wordpress-mobile/WordPress-iOS
# Stories: Fix Media Import bugs

* Fixes a bug where media was unnecessarily exported (instead of copied/reused) when adding to story.
* Adds a Loading indicator during media import so there is some indicator while media files are being exported.
* Allows video uploads for local URLs so that Stories media can be uploaded to the media library.

### Testing
* Go to the Blog Details page
* Create a new Story Post
* Add several pieces of media (with at least 1 video) from your device
* Ensure that media is imported quickly into the Stories editor
* Publish the post
* Ensure that video is uploaded and published in story

## Code Overview



PR submission checklist:
